src/Model/Document.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the jolicode/elastically library.
+ *
+ * (c) JoliCode <coucou@jolicode.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JoliCode\Elastically\Model;
+
+use Elastica\Document as ElasticaDocument;
+
+class Document extends ElasticaDocument
+{
+    private ?object $dto = null;
+
+    public function __construct($id, object $dto = null, $data = [], $index = '')
```

Comment:
WDYT about enforcing the type on data and index? (put the same as in the PHPDoc of ElasticaDocument?)

Reply:
Sure